### PR TITLE
Trying to pull it !

### PR DESCRIPTION
**Pull request by Icuisset**
Cohort 11 Guestbook
@Isabelle Cuisset was here!

### CODE REVIEW

**Comment by Icuisset** [comment]
Hi !
I tried to pull on my own fork first and now on the master...
Not sure this is working ! :-)

File: README.md [hunk 1]
```diff
@@ -2,4 +2,7 @@
 
 ## Want to see it live?
 
-View it [here](http://near-drink.surge.sh/)!
\ No newline at end of file
+View it [here](http://near-drink.surge.sh/)!
+
+Cohort 11 Guestbook
```
Comment (User5842) [inline README.md:7]: Nice work! How can we make this a heading instead? Consider using a first-level heading.

Comment (Icuisset) [reply README.md:7]: OK, let me try this !!! :-)

**Review by User5842 — APPROVED** [review]

**Comment by Icuisset** [comment]
Aaah, thank you !! 😊

**Pull request merged**